src/sparseml/evaluation/cli.py

```diff
@@ -170,27 +175,66 @@ def main(
         )
 
 
-def _args_to_dict(args: Tuple[Any, ...]) -> Dict[str, Any]:
```

Comment:
you do not to duplicate this, you have this function already in sparseml.utils. somewhere somwehere

Reply:
Thanks!